docs/howto/writing-migrations.txt

```diff
@@ -103,6 +103,8 @@ the respective field according to your needs.
     :filename: 0006_remove_uuid_null.py
 
     # -*- coding: utf-8 -*-
+    # Generated by Django 1.9
```

Comment:
I don't know that we need these comments in the documentation, especially since the dates will go stale (note that you can use |version| to get the Django version dynamically).